hw/ip/gpio/dv/cov/gpio_cov_excl.el

```diff
@@ -1,34 +1,55 @@
-// Copyright lowRISC contributors (OpenTitan project).
-// Licensed under the Apache License, Version 2.0, see LICENSE for details.
-// SPDX-License-Identifier: Apache-2.0
 //==================================================
 // This file contains the Excluded objects
-// Generated By User: gaurangg
+// Generated By User: marcelo
 // Format Version: 2
-// Date: Wed Oct 30 17:22:12 2019
+// Date: Fri Feb 14 17:12:55 2025
 // ExclMode: default
 //==================================================
-CHECKSUM: "3109872245 3727953184"
-INSTANCE: tb.dut.u_reg.u_data_in
-ANNOTATION: "[UNR]"
-Block 1 "0" "assign wr_en = de;"
-CHECKSUM: "1352181506 3803838024"
+CHECKSUM: "1706182284 132761700"
+INSTANCE: tb.dut.u_reg.u_reg_if.u_rsp_intg_gen
+ANNOTATION: "The parameter EnableRspIntgGen is zero, so the condition will never hit."
+Block 1 "461445014" "assign rsp_intg = tl_i.d_user.rsp_intg;"
+ANNOTATION: "The parameter EnableRspIntgGen is zero, so the condition will never hit."
+Block 2 "2643129081" "assign data_intg = tl_i.d_user.data_intg;"
+CHECKSUM: "2099741489 2073313596"
+INSTANCE: tb.dut.u_reg.u_data_in.wr_en_data_arb
+ANNOTATION: "Becasuse of the data_in is a read-only register, the signals wr_en and wr_data are not applicable. "
```

Comment:
This annotation is wrong. The missing line coverage means that `prim_subreg_arb.sv`, line 43 never excutes. Again, this is a continuous assignment, so it really means that `de` is constant. Here, `hw2reg.data_in.de` is wired to `1` (`gpio.sv`, line 112).

And that's fine: the hardware updates `data_in` on every cycle.

But the annotation needs fixing.

Reply:
This looks more sensible, thanks. I'd probably suggest tweaking it slightly to avoid line numbers. Otherwise unrelated changes that e.g. add a line above will make this annotation false.